src/output.F90

```diff
@@ -356,7 +356,7 @@ subroutine print_batch_keff()
     integer :: n  ! number of active generations
 
     ! Determine overall generation and number of active generations
-    i = overall_generation()
+    i = overall_generation() - 1
```

Comment:
Rather than relying on overall_generation here, we should probably just make this:
```Fortran
i = current_batch*gen_per_batch
```

Reply:
Thanks @paulromano, pushed some changes without realizing they would update the PR as well. I'm still in the middle of figuring out some issues with travis but will push some more commits once that is done.

Reply:
Yeah, I noticed a few tests were failing. The change I'm suggesting here should hopefully fix those tests.